inspirehep/modules/records/jsonschemas/records/experiments.json

```diff
@@ -142,6 +138,10 @@
                         ],
                         "title": "Type of relation",
                         "type": "string"
+                    },
+                    "record": {
+                        "title": "URI for the related experiment record",
```

Comment:
I see the schemas were using `title` and `description` in non consistent ways. We should standardize this. In principle `title` would be used as a label by a potential JSON-Editor, while `description` is more the documentation. So `title` is actually more important than `description`. Can you standardize to that?